device/config/functions.go

```diff
@@ -66,10 +66,32 @@ func SaveLocale(storage map[string]interface{}) error {
 
 // SetKeyboard is a default method to with dialog to configure the keymap
 func SetKeyboard(storage map[string]interface{}) error {
+	var (
+		locale string
+		ok     bool
+	)
+
+	if locale, ok = storage[GetConstLiteral(Locale)].(string); ok {
+		if i := strings.IndexAny(locale, "_."); i >= 0 {
+			locale = locale[:i]
+		}
+	}
+
+	fmt.Println("[+] Default keyboard layout: ", constants.DefaultKeymap)
+
+	if dialogs.YesNoDialog("Change default keyboard layout?") {
+		inp := dialogs.GetSingleAnswer("New keyboard layout: ",
+			dialogs.CreateValidatorFn(func(layout string) error { return constants.ValidateLayout(locale, layout) }))
+
+		arr := constants.GetLayout(locale, inp)
```

Comment:
__[gometalinter]__
* [unconvert] GetLayout not declared by package constants
* [gosimple] GetLayout not declared by package constants
* [staticcheck] GetLayout not declared by package constants

<!-- Comment by [SideCI](https://sideci.com) -->